biogrid/apps/api/src/app/services/Biogrid.ts

```diff
@@ -11,21 +18,46 @@ export interface NewBiogridOpts {
   largeBatteryCells: number;
 }
 
+// TODO change to a stateless solution
+// See issue: https://github.com/googleinterns/step141-2020/issues/50
+let biogrid: Biogrid;
+const biobrain = BioBrain.Instance;
+const states: any[] = [];
+
+// TODO, allow users to specify number of buildings, town size, and number of Solar Panels
+// See issue: https://github.com/googleinterns/step141-2020/issues/49
 export async function createNewBiogrid(body: NewBiogridOpts) {
-  // TODO implement
-  return "Fake"
+  const buildings = [
+    new Building(10, 2, 3),
+    new Building(10, 5, 4),
+    new Building(10, 4, 3),
+    new Building(10, 1, 2),
+    new Building(10, 3, 1),
+  ];
+  const town = new RuralArea(buildings, 10, 10);
+  biogrid = new Biogrid(town, {
+    numberOfLargeBatteryCells: body.largeBatteryCells,
+    numberOfSmallBatteryCells: body.smallBatteryCells,
+    numberOfSolarPanels: 10,
+  });
+  return 'Created';
```

Comment:
are there cases when the Biogrid is not created? if so, what is returned? If not, do you have to return this?

Reply:
Yeah, it should throw if it is not created. Error handling is automatically handled by Tsoa then